pennylane/labs/phase_polynomials/parity_matrix.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Parity matrix representation"""
+
+from typing import Iterable
+
+import numpy as np
+
+import pennylane as qml
+
+
+def parity_matrix(circ: qml.tape.QuantumScript, wire_order: Iterable = None):
+    r"""
+    :doc:`Parity matrix intermediate representation <compilation/parity-matrix-intermediate-representation>` of a CNOT circuit
```

Comment:
Maybe?
```suggestion
    r"""Compute the
    :doc:`parity matrix intermediate representation <compilation/parity-matrix-intermediate-representation>` of a CNOT circuit.
```

Not sure about the capitalization of "parity matrix" mid-sentence...